### DISCUSSION TITLE
KEDA Chart v2.13.2

### DISCUSSION THREAD

**Discussion by JorTurFer** [kedacore/charts#615] [Announcements]
Chart: v2.13.1 | App: v2.13.0

## TL;DR

```shell
helm repo add kedacore https://kedacore.github.io/charts
helm repo update
helm install keda kedacore/keda --version 2.13.2 -n keda --create-namespace
```

### Fixes / Changes
- Support to KEDA v2.13.1
- Several fixes related with prometheus integrations

## What's Changed
* fix: pod monitor port name does not match deployment spec by @pragmaticivan in https://github.com/kedacore/charts/pull/602
* feat: Add support for metricRelabelings to Pod and Service monitors by @pragmaticivan in https://github.com/kedacore/charts/pull/603
* feat(keda): Allow using own Cert-manager Issuer/ClusterIssuer for issuing KEDA TLS certificates by @zerodayyy in https://github.com/kedacore/charts/pull/530
* Allow extra egress rules for the keda operator ciliumnetworkpolicy by @QuentinBisson in https://github.com/kedacore/charts/pull/610
* chore: Add missing comment to values.yaml by @JorTurFer in https://github.com/kedacore/charts/pull/611

## New Contributors
* @pragmaticivan made their first contribution in https://github.com/kedacore/charts/pull/602

**Full Changelog**: https://github.com/kedacore/charts/compare/v2.13.1...v2.13.2

<hr /><em>This discussion was created from the release <a href='https://github.com/kedacore/charts/releases/tag/v2.13.2'>KEDA Chart v2.13.2</a>.</em>
